cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -804,3 +807,21 @@ def _remove_cluster(node, logger):
 
     # yum clean all doesn't clean all, so let's be more forceful
     node.run_command('sudo rm -rf /var/cache/yum')
+
+
+def _restore_from_xfs(nodes, logger):
```

Comment:
I think these two also need to be run on the function/session scoped VM combo in cosmo_tester.conftest?